### PR TITLE
Added bibtex example

### PR DESCRIPTION
**Pull request by pboothe**
This change is [Reviewable](https://reviewable.io/reviews/m-lab/website/514)

### CODE REVIEW

File: _pages/04-data.md [hunk 1]
```diff
@@ -115,3 +115,19 @@ The M-Lab *test name* Data Set, *date range used*. *M-Lab test URL*
 For example:
 
 The M-Lab NDT Data Set 2009-02-11–2015-12-21. [https://measurementlab.net/tests/ndt]({{ site.baseurl }}/tests/ndt)
+
+or, in BibTeX:
```
Comment (critzo) [inline _pages/04-data.md]: Could this say "BibTeX format" and be a link to a resource about BibTex?

Comment (pboothe) [reply _pages/04-data.md]: "BibTeX format" will definitely make that change.

As far as a link to "What is BibTeX?", I'd probably just link to Wikipedia, which feels a little like saying "look it up yourself if you are confused".

**Comment by pboothe** [comment]
Added to the NDT page as well and changed to BibTeX format. If you think a wikipedia link is useful, let me know and I will add it.

File: _pages/04-data.md (hunk 1 above)
Comment (critzo) [reply _pages/04-data.md]: that's what I usually do- link to wikipedia. which isn't presumptive, but helpful in saving the person who would need to ask that from needing to do the search.

File: _pages/04-data.md [hunk 2]
```diff
@@ -115,3 +115,19 @@ The M-Lab *test name* Data Set, *date range used*. *M-Lab test URL*
 For example:
 
 The M-Lab NDT Data Set 2009-02-11–2015-12-21. [https://measurementlab.net/tests/ndt]({{ site.baseurl }}/tests/ndt)
+
+or, in [BibTeX](https://en.wikipedia.org/wiki/BibTeX) format:
```
Comment (critzo) [inline _pages/04-data.md]: Last one :) - can we add the new window target code to this link: `[BibTeX](https://en.wikipedia.org/wiki/BibTeX){:target="_blank"}`

Comment (critzo) [reply _pages/04-data.md]: And one more correction identified by the linter:
`_pages/tests/ndt/ndt.md:55: MD012 Multiple consecutive blank lines`

**Pull request merged**